### DISCUSSION TITLE
Is there some restriction of any time on some geographical areas?

### DISCUSSION THREAD

**Discussion by micheledp93** [Leaflet/Leaflet#9272] [Q&A] [answered]
Hi, my company is developing ann application using Leaflet. They asked me to check is there are some limitations of some kind like on some areas of the maps. For example, if there are some counties or some particular areas that are "obscured"? I know it's a strage question, but I was told to ask so... 😂

**Comment by Falke-Design** [comment, marked as answer]
No there are no restriction but to clarify, we don't provide any map data, we are only make it possible to show them.

PS: Feel free to say your boss, that you used now your "one time free stupied question" coupon. I was happy to help you in my unpaid free time. Next time I only will answer if I'm getting paid ;)
